### ISSUE TITLE
VZ Polygons Checklist

### ISSUE THREAD

**Issue by alan-deanda** [cityofaustin/atd-data-tech#8388]
# Polygon maintenance checklist

## Completed, reviewed polygons

- [x] 1727C0E486: Lewis asked this one to be adjusted at least to capture the slip lane

- [x] E3A6F25A01: doesn't include the legs and stems from being on the edge of the ASMP network, perhaps?
  **Comment:** This would be fixed more cleanly by digitizing the on-ramps than by attempting to modify the existing polygon - we don't have a clear way to define the scope of such an edit.

- [x] 4C21F93901: E SH 71 EB TO S FM 973, S FM 973 RD

- [x] 7259D99593: This location does not capture the slip lane (right turn).

- [x] Missing portions of US 290 WB @ Joe Tanner Ln
  **Comment** This was mostly resolved by adding street level 5 polygons

- [ ] #9182
  ![image](https://user-images.githubusercontent.com/3653206/135141243-421b928b-a569-4df5-8eb8-c1ac2514b063.png)

  **Comment** This is a complex edit. Either the entire section gets re-created, or the polygon is split and edited to fit. New polygons would likely still need to be created.

- [ ] #9183

  **Comment** We need a workflow for adding new segments to guarantee unique IDs. Several Polygons need to be either deleted or re-purposed. Which is preferable?

- [x] #9184

  **Comment** We need a workflow for combining segments/assigning new IDs

- [x] #9185

  **Comment** While this polygon is bad, it's unclear how to fix the geometry, as  it's really bad at the entire intersection. Polygons were built around terribly misaligned street lines and there is a lot of missing geometry for key sections, like u-turns and slip lanes and entire frontage roads.

- [ ] #9143
**Comment** The CTN is digitized to show the centerline of a road, not the "sausage links" of split polygons

- [x] Manually draw E3A6F25A01 (US 183 and Loyola) to better represent the intersection post-183 South construction.
**Comment** This edit is made with the limited scope of making this polygon cover the intersection. We need new polygons to cover the frontage roads for more proper coverage.

- [x] add E/W legs to N Lamar/5th (D0A613CEA8) 

- [x] Clean up the edge cases of these polygons @ W. Louis Henna Blvd & CR 172 (B569606A44, 2A9F310D05)

- [x] E3A6F25A01: Per Lewis, Can you please add this to the list of locations that need the polygon adjusted to add in approaches on service roads? Need to split up SB intersection and NB intersection into two distinct polygons too. Lots of construction in this area lately which I think is complete now.

- [x] Review polygons for Springdale & Webberville, in particular regarding the slip-lane

- [x] Stassney (just east of Mira Dr, split lane is not wide wide enough)
- [x] Missing Roadway at Lamar & Koenig (Right Turn Cut-Through)
- [x] Montopolis (just north of Oltorf the split lane is not wide enough)

## Alan's polygons

- [x] #9186

- [ ] #9187

- [x] #9188

## Adrien's polygons

- [x] #9189

- [ ] #9190

- [ ] #9191

- [x] #9192

- [x] #9193

- [x] #9194

**Comment by joelmeyer21** [comment]
Please double check Pearce Lane in SE Austin. We have polygons here that I believe actually fall outside the Full Purpose Jurisdiction.

**Comment by johnclary** [comment]
thanks @joelmeyer21. @alan-deanda will be re-focusing on this work in the next two weeks. please let us know if any specific areas are a priority.

**Comment by patrickm02L** [comment]
Hey @adrien-atd, here's the Issue that we're using to track all the polygons that need work on them.

**Comment by adrien-atd** [comment]
Thanks @patrickm02L !

**Comment by patrickm02L** [comment]
Here's the notes from our meeting on 1/16/22
* How to divvy up the ones that still have to be completed?
* What are the polygons that are being blocked by CTN?
* What are the polygons that are one polygon that needs to be split up into multiple?
* What are the polygons that come from multiple and need to be merged into one?
* What to get clarification from Lewis about
    * Metric Boulevard
    * Montopolis and 183 (considerably different)
* What to do about private drives?
    * Oldham St

**Comment by patrickm02L** [comment]
@alan-deanda Create label. Project - VZ Polygon Updates

**Comment by alan-deanda** [comment]
New issues have been created for all outstanding requests in this checklist.

**Comment by patrickm02L** [comment]
@patrickm02L bring this up in the next GIS Sprint meeting Tuesday 8/23/22.

**Comment by patrickm02L** [comment]
Both Alan and Adrien, who were using this epic to track VZ polygons updates, have departed from DTS and VZ. 

While there are still polygons to be adjusted, we've come up with a process for VZ GIS staff to edit future VZ polygons on MAINT (which is documented in #11003) that will serve as a more efficient and simpler process. 

Therefore, I am closing this epic and the subsequent Issues.
